FreeRTOS/FreeRTOS
# Move PKCS #11 Mutual Auth Demo from LTS Dev branch

Move PKCS #11 Mutual Auth Demo from LTS Dev branch

Description
-----------
This change moves the PKCS #11 Mutual Auth demo. The demo has previously been reviewed, but the project needed to be updated to address changes to the folder structure.

Test Steps
-----------
Follow the demo instructions on freertos.org for the PKCS #11 mutual auth demo, and connect to AWS IoT Core.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
